test.js

```diff
@@ -0,0 +1,36 @@
+const test = require('tape')
+const noNodeConfig = require('./index')
+const co = require('co')
+const config = require('./testConfig/configData')
+const keys = ['mongo', 'redis']
+const PROCESS_ENV = process.env.NODE_ENV = 'test_env'
+
+const configObject = co(function* () {
```

Comment:
@anuragCES I don't see the point to have it. Adding 3 lines to wrap one line. Why not to use
```js
noNodeConfig({config}).then(...
```
or
```js
let result = yield noNodeConfig({config})
```
directly in the test cases.